css/team.css

```diff
@@ -58,14 +61,16 @@
 
 .social {
     position: absolute;
-    bottom: -18%;
-    left: 42%;
+    bottom : -24%;
+    left:46%;
+    /* bottom: -18%;
+    left: 42%; */
 }
 
 .social img {
     padding: 0 15%;
     width: 25px;
-    height: 25px;
+    height: 20px;
```

Comment:
Is the component where this css has been applied responsive? `%` might work better than `px` or  `@media` query if it is not responsive.